src/ssl.c

```diff
@@ -41022,6 +41022,16 @@ int  wolfSSL_get_chain_cert_pem(WOLFSSL_X509_CHAIN* chain, int idx,
 #endif /* WOLFSSL_PEM_TO_DER || WOLFSSL_DER_TO_PEM */
 }
 
+/* get session IDSz */
```

Comment:
Can we add a comment here to indicate `SSL_MAX_SSL_SESSION_ID_LENGTH` is the public enum for max session ID length?